packages/flutter_tools/lib/src/ios/devices.dart

```diff
@@ -396,12 +396,23 @@ class _IOSDeviceLogReader extends DeviceLogReader {
     });
   }
 
+  /// Blacklist of log output lines to suppress.
+  ///
+  /// Log lines begginning with strings in this list are suppressed from iOS
+  /// device log output.
+  static const List<String> _logBlacklist = const <String>[
```

Comment:
nit: do we still start constant names with `_k`? Also, since this constant is used in exactly one place, I'd make it a local `const` variable.